boards/sim/sim/sim/scripts/Make.defs

```diff
@@ -63,6 +63,11 @@ ARCHINCLUDES = -I. -isystem $(TOPDIR)/include
 ARCHINCLUDESXX = -I. -isystem $(TOPDIR)/include -isystem $(TOPDIR)/include/cxx
 ARCHSCRIPT =
 
+ifneq ($(KERNEL),y)
+  ARCHCPUFLAGS += -fno-common
+  ARCHCPUFLAGSXX += -fno-common
+endif
```

Comment:
need add to other Mak.defs under boards/sim/sim/sim/configs

Reply:
probably.
the configs involving other make.defs don't build on macOS for unrelated reasons though.

Reply:
But other people may try to build that config, it's better to fix all together.

Reply:
ok